### PR TITLE
[YOCTO] Modified unittest directory for installation & runn @open sesame 04/01 10:54

### PR DESCRIPTION
**Pull request by jijoongmoon**
# PR Description
Because there are test cases incluiding absolute path for library, it
is hard to run unit testcases after installation. In this PR, these
testcases and nnstreamer_example libraries are installed
/usr/lib/nnstreamer/unittest. For the nnstreamer_example, libraries
are in /usr/lib/nnstreamer/unittest/nnstreamer_exampls, and For the
tests, they are installed in /usr/lib/nnstreamer/unittest/tests.
Some unittest executions (unittest_repo, unittest_sink,
unittest_common, unittest_plugins) are installed in
/usr/lib/nnstreamer/unittest.

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

**Comment by myungjoo** [comment]
Because you have modified install paths, you need to update .spec (rpm) and .install (deb) files.

File: tests/meson.build [hunk 1]
```diff
@@ -82,7 +90,8 @@ if gtest_dep.found()
   unittest_plugins = executable('unittest_plugins',
     join_paths('nnstreamer_plugins', 'unittest_plugins.cpp'),
     dependencies: [nnstreamer_unittest_deps],
-    install: false
+    install: install_test,
```
Comment (kparichay) [inline tests/meson.build]: update this for `unittest_src_iio` as well? (line 103)

Comment (jijoongmoon) [reply tests/meson.build]: Oh.. I didn't aware of that. I will check it and update it if needed. Thanks.

File: tests/nnstreamer_filter_custom/runTest.sh [hunk 2]
```diff
@@ -34,7 +34,12 @@ gstTest "videotestsrc num-buffers=1 ! video/x-raw,format=RGB,width=280,height=40
 
 
 # Test constant passthrough custom filter (1, 2)
-PATH_TO_MODEL="../../build/nnstreamer_example/custom_example_passthrough/libnnstreamer_customfilter_passthrough.so"
+if [[ -z "${CUSTOMLIB_DIR// }" ]]
```
Comment (leemgs) [inline tests/nnstreamer_filter_custom/runTest.sh]: Why do we use "//" instead of "/"?

Comment (jijoongmoon) [reply tests/nnstreamer_filter_custom/runTest.sh]: so that remove ${CUSTOMLIB_DIR}=" ".

File: tests/meson.build (hunk 1 above)
Comment (jijoongmoon) [reply tests/meson.build]: Done.

**Comment by myungjoo** [comment]
PTAL: @leemgs @kparichay

**Pull request merged**